utils/pvrtex/pvr_texture_encoder.c

```diff
@@ -838,6 +838,7 @@ void pteAutoSelectPixelFormat(PvrTexEncoder *pte) {
 	assert(pte->src_img_cnt);
 
 	unsigned clearpix = 0, halfpix = 0, opaquepix = 0;
+	(void)opaquepix;
```

Comment:
Why not just remove this entirely? I guess it could be nice for documentation but could just have the inner loop if block have `else if (a != 0xff) halfpix++;` yea?

Reply:
Sure could. I dunno. I was like "maybe one day TapamN wants to use this variable, or he wouldn't have had it?" Dunno. Can just as easily remove it. Will do, np.